installer.sh

```diff
@@ -249,12 +249,16 @@ install_debs(){
   if ! command -v gcc > /dev/null ; then
     apt-get install -y build-essential
   fi
-  # Docker + docker-compose
+  # Docker + docker compose
   if [[ "${WSL}" -eq 1 ]]; then
     echo -e "\n${ORANGE}WARNING: If you are using WSL2, disable docker integration from the docker-desktop daemon!${NC}"
     read -p "Fix docker stuff, then continue. Press any key to continue ..." -n1 -s -r
   fi
-  if ! command -v docker > /dev/null || ! command -v docker-compose > /dev/null ; then
+  if ! command -v docker-compose > /dev/null;
```

Comment:
are we running into issues with the EMBA installer here? EMBA uses by default the old variant